yb-voyager/cmd/root.go

```diff
@@ -283,37 +288,16 @@ func registerCommonGlobalFlags(cmd *cobra.Command) {
 
 	BoolVar(cmd.Flags(), &callhome.SendDiagnostics, "send-diagnostics", true,
 		"enable or disable the 'send-diagnostics' feature that sends analytics data to YugabyteDB.(default true)")
+
+	cmd.PersistentFlags().StringVarP(&cfgFile, "config-file", "c", "",
```

Comment:
do you want to hide this now? once we have sufficient tests for each command, we can enable it one by one.